tests/Makefile

```diff
@@ -181,7 +181,7 @@ ifdef RECORD_PSA_STATUS_COVERAGE_LOG
 # Explicitly depend on this header because on a clean copy of the source tree,
 # it doesn't exist yet and must be generated as part of the build, and
 # therefore the wildcard enumeration above doesn't include it.
-TEST_OBJS_DEPS += include/test/instrument_record_status.h
+TEST_OBJS_DEPS += ../framework/tests/include/test/instrument_record_status.h
```

Comment:
What about `instrument_record_status.h` in lines 270 and 287?

Reply:
Good catch, thankyou!

Reply:
Note that the record status feature is something we use rarely, only for investigation purposes, so it's not the end of the world if it breaks. Also, we have a path to [replace it using the new wrapper generator](https://github.com/Mbed-TLS/mbedtls/pull/8559)). So if it helps, feel free to remove it.